cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/StreamAdmin.java

```diff
@@ -109,7 +110,7 @@
    * Register stream used by program.
    *
    * @param streamId the stream being used
-   * @param programId the program that is using the stream
+   * @param owners the ids that are using the stream
    */
-  public void register(Id.Stream streamId, Id.Program programId);
+  public void register(Id.Stream streamId, List<Id> owners);
```

Comment:
same nit as above. use the same pattern, i don't think the order matters, but lets follow one pattern